web/beacon-app/src/components/auth/LandingHeader/LandingHeader.tsx

```diff
@@ -2,21 +2,23 @@ import { AriaButton as Button } from '@rotational/beacon-core';
 import { memo } from 'react';
```

Comment:
@elysee15  this component is used by 2 other features components. would you mind moving it to the Components folder